src/PKSim.Core/Model/RandomPopulationFactory.cs

```diff
@@ -75,11 +90,12 @@ public Task<RandomPopulation> CreateFor(RandomPopulationSettings populationSetti
                fllUpGenderQueueBasedOn(populationSettings);
                progressUpdater.Initialize(populationSettings.NumberOfIndividuals, PKSimConstants.UI.CreatingPopulation);
 
+               var diseaseStateImplementation = _diseaseStateImplementationFactory.CreateFor(populationSettings.BaseIndividual);
                //the base individual is used to retrieve the default values. 
-               var baseIndividual = populationSettings.BaseIndividual;
+               var baseIndividual = diseaseStateImplementation.CreateBaseIndividualForPopulation(populationSettings.BaseIndividual);
```

Comment:
First hook that we discussed. Return the base individual for the population. I am not sure how generic this will be